internal/handlers/publishers.go

```diff
@@ -0,0 +1,99 @@
+package handlers
+
+import (
+	"errors"
+
+	"github.com/gofiber/fiber/v2"
+	"gorm.io/gorm"
+
+	db "github.com/italia/developers-italia-api/internal/database"
+	"github.com/italia/developers-italia-api/internal/models"
+)
+
+func GetPublishers(ctx *fiber.Ctx) error {
+	var publishers []models.Publisher
+
+	db.Database.Find(&publishers)
```

Comment:
Magari il find possiamo farlo sul modello, così ci possiamo aggiungere del log sulle varie operazioni (tipo gli passi l'orm, fa la query e logga anche)
Che ne pensi?
Se vuoi propongo io la modifica

Reply:
Good, I also like this approach. Let's use hooks if needed :-)